html/janus.js

```diff
@@ -125,6 +127,33 @@ Janus.init = function(options) {
 				callback([]);
 			}
 		}
+		// Helper methods to attach/reattach a stream to a video element (previously part of adapter.js)
+		Janus.attachMediaStream = function(element, stream) {
+			if(adapter.browserDetails.browser === 'chrome') {
+				var chromever = adapter.browserDetails.version;
+				if(chromever >= 43) {
+					element.srcObject = stream;
+				} else if(typeof element.src !== 'undefined') {
+					element.src = URL.createObjectURL(stream);
+				} else {
+					Janus.error("Error attaching stream to element");
+				}
+			} else {
+				element.srcObject = stream;
+			}
+		};
+		Janus.reattachMediaStream = function(to, from) {
+			if(adapter.browserDetails.browser === 'chrome') {
+				var chromever = adapter.browserDetails.version;
+				if(chromever >= 43) {
+					to.srcObject = from.srcObject;
+				} else if(typeof element.src !== 'undefined') {
+					to.src = from.src;
+				}
+			} else {
+				to.srcObject = from.srcObject;
+			}
+		};
 		// Detect tab close
 		window.onbeforeunload = function() {
```

Comment:
I see the point you make, that might indeed cause trouble. I guess this can be an option. For instance, we already have a `destroyOnUnload`, that dictates whether you should automatically destroy a new session on unload. We might add an option to the `init` where you specify the behaviour in that regard, e.g.:

1. just set the `unbeforeunload`, as we do now;
2. set the `unbeforeunload` and invoke the previous one at the end;
3. don't set the `unbeforeunload` at all (which makes `destroyOnUnload` useless).